app/src/main/kotlin/com/wire/android/ui/home/conversationslist/bottomsheet/ConversationSheetContent.kt

```diff
@@ -167,7 +167,7 @@ sealed class ModalSheetContent(val title: String, val conversationId: Conversati
 
     class GroupConversationEdit(
         title: String,
-        val groupColorValue: Long,
+        val groupColorValue: ULong,
```

Comment:
Maybe we could start using `Color` instead of `ULong`? Because right now it's taken from `Color` (`getConversationColor({id}).value`) and finally converted back to `Color` (`Color(groupColorValue)`) 😅